contrib/babelfishpg_tsql/sql/sys_functions.sql

```diff
@@ -1136,6 +1136,10 @@ create or replace function sys.babelfish_timezone_mapping(IN tmz text) returns t
 AS 'babelfishpg_tsql', 'timezone_mapping'
 LANGUAGE C IMMUTABLE ;
 
+create or replace function sys.timezone_mapping_pg_to_windows(IN tmz text) returns text
+AS 'babelfishpg_tsql', 'timezone_mapping_pg_to_windows'
+LANGUAGE C IMMUTABLE PARALLEL SAFE STRICT;
```

Comment:
This is a internal helper function, so lets prefix it with babelfish: `babelfish_timezone_mapping_pg_to_windows `